### ISSUE TITLE
Submit APK in release

### ISSUE THREAD

**Issue by TheEvilSkeleton** [MuntashirAkon/Metro#75]
**Is your feature request related to a problem? Please describe.**
There's no easy way to get the latest release of Metro at the moment.

**Describe the solution you'd like**
I suggest submitting a release with the APK on GitHub. I know the release says that it will get to F-Droid in a couple of days, but I didn't expect it to be *that* long.

At lease for those who want the latest version can get the latest version for the time being.

**Comment by MuntashirAkon** [comment]
Now available in F-Droid.

**Comment by noClaps** [comment]
You should submit APK releases on GitHub anyway, for those who don't want to / can't use F-Droid.

**Comment by soopstar** [comment]
I agree. From my experience, I can get an update faster from apps than on F-Droid

**Comment by MuntashirAkon** [comment]
Those who are not aware of the previous announcement: Metro was discontinued as there were no upstream activities. But it has changed a few months ago when the upstream became active again. So, Metro is now in the low maintenance mode. I have made some extra changes such as making it fully offline as it was promised or changing the font-sizes which looked disproportional. But in the end, I do not use the app myself, and I _only_ maintain this app because I thought F-Droid community is missing out a good open source app and the requests that I receive via E-Mail. Therefore, I do not see any reason to publish it outside of F-Droid. If you're interested in GitHub releases, you can [download](https://github.com/RetroMusicPlayer/RetroMusicPlayer/releases/latest) the app from the upstream instead.

> I agree. From my experience, I can get an update faster from apps than on F-Droid

You get updates once or twice in a year. Waiting for two/three days isn't a great loss.

**Comment by TheEvilSkeleton** [comment]
That's a fair point. 

> But in the end, I do not use the app myself, and I _only_ maintain this app because I thought F-Droid community is missing out a good open source app and the requests that I receive via E-Mail.

About this, I thank you so much for your valuable service to the F-Droid community! I use Metro regularly and I'm glad it exists. It's my favorite music player on Android.
